source/widgets/visualize/freeviz.md

```diff
@@ -16,40 +16,44 @@ Displays FreeViz projection.
+   - Set [*jittering*](https://en.wikipedia.org/wiki/Jitter) to prevent the dots from overlapping (especially for discrete attributes).
+   - Anchors inside a circle are hidden. Circle radius can be changed using *Hide radius*.
+   - *Show color regions* colors the graph by the *Color* attribute (see the screenshot below).
    - *Show legend* displays a legend on the right. Click and drag the legend to move it.
-   - *Show class density* colors the graph by class (see the screenshot below).
-   - *Label only selected points* allows you to select individual data instances and label them.
-5. *Select, zoom, pan and zoom to fit* are the options for exploring the graph. The manual selection of data instances works as an angular/square selection tool. Double click to move the projection. Scroll in or out for zoom.
-6. If *Send automatically* is ticked, changes are communicated automatically. Alternatively, press *Send*.
-7. *Save Image* saves the created image to your computer in a .svg or .png format.
-8. Produce a report.
+
+4. *Select*, *zoom*, *pan* and *zoom to fit* are the options for exploring the graph. The manual selection of data instances works as an angular/square selection tool. Scroll in or out for zoom.
+5. If *Send automatically* is ticked, changes are communicated automatically. Alternatively, press *Send*.
 
 Manually move anchors
 ---------------------
 
-![](images/freeviz-moveanchor.png)
+![](images/FreeViz-anchors.png)
 
 One can manually move anchors. Use a mouse pointer and hover above the end of an anchor. Click the left button and then you can move selected anchor where ever you want.
 
 Selection
 ---------
 
-Selection can be used to manually defined subgroups in the data. Use Shift modifier when selecting data instances to put them into a new group. Shift + Ctrl (or Shift + Cmd on macOs) appends instances to the last group.
+Selection can be used to manually define the subgroups in the data. The default tool is *Select*, which selects data instances within the chosen rectangular area. Use Shift modifier when selecting data instances to put them into a new group. Shift + Ctrl (or Shift + Cmd on macOs) appends instances to the last group. Alt removes the chosen instance from the group.
 
-Signal data outputs a data table with an additional column that contains group indices.
+*Pan* enables you to move the plot around the pane. With *Zoom* you can zoom in and out of the pane with a mouse scroll, while *Reset zoom* resets the visualization to its optimal size.
+
+The widget outputs a data table with an additional column that contains group indices.
 
 ![](images/FreeViz-selection.png)
 
-Explorative Data Analysis
--------------------------
+Example
+-------
+
+An example of a simple schema, where we observe a projection of the *zoo* dataset. We optimized the plot to place similar points together. Thus we can observe that mammals are distinctly different from other animals.
```

Comment:
Next one: we used to use the word "schema", but now refer to it as a "workflow".